### DISCUSSION TITLE
Welcome to starlight-zen-mode Discussions!

### DISCUSSION THREAD

**Discussion by trueberryless** [trueberryless-org/starlight-view-modes#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by robmunger** [comment]
Thanks! I spend an awful lot of time in Starlight and love the idea of being able to customize display modes.

I actually use Starlight for teaching; I share notes in economics and finance with my students via a collection of sites. Students love how fast it is and that they can use it on both computers and mobile devices with no installs.

Overall, I like the ability to trim down some of the chrome in Starlight. Being able to hide or slim down the bar at the top or the sidebars would make it more ergonomic to engage deeply with the main content. [The pull request about collapsible side panels](https://github.com/trueberryless/starlight-view-modes/pull/10) looks potentially very appealing, if it works with your vision for the extension.

**Reply by trueberryless** [reply]
First of all: Thanks for using the plugin!

The original idea of this plugin (and still part of my philosophy) is using the plugin for teaching purposes so I'm really honoured that you use it exactly in this case.

My plan is to add the presentation mode in the future, which would be soooooo cool for presentations in any kind of education. Unfortunately I currently do not have that much time for programming on this repo. Moreover the ground concept of how to implement a presentation mode must be overthought by enthusiastic programmers as well (I'll maybe start a discussion on this one).

I'm going to look into the PR hopefully soon, I'm glad to see developments from the Starlight community here, thx u all ❤️!

**Reply by robmunger** [reply]
Thanks for writing the plugin!!

The entire Starlight ecosystem is really wonderful for teaching. I also have a software platform called Canvas available (https://www.instructure.com/canvas), and I far prefer Starlight for actual, in-depth content. It would likewise be terrific for online books.
